benchmarks/tpch/justfile

```diff
@@ -0,0 +1,52 @@
+set fallback
+VENV := ".venv"
+VENV_BIN := ".venv/bin"
+
+
+@venv:
+	@python3 -m pip install virtualenv > /dev/null
+	@python3 -m virtualenv {{VENV}} > /dev/null
+
+## Set up virtual environment and install requirements
+@requirements: venv
+	{{VENV_BIN}}/python -m pip install --upgrade pip > /dev/null
+	{{VENV_BIN}}/pip install -r requirements.txt > /dev/null
+	
+clean-tpch-dbgen:
+	make -C tpch-dbgen clean
+
+clean-venv:
+	rm -r .venv
+
+clean-tables:
+	rm -r tables_scale_*
+
+clean: clean-tpch-dbgen clean-venv
+
+gen_scale_factor n: requirements
+	make -C tpch-dbgen all
+	cd tpch-dbgen && ./dbgen -vf -s {{n}} && cd ..
+	mkdir -p "tables_scale_{{n}}"
+	mv tpch-dbgen/*.tbl tables_scale_{{n}}/
+	{{VENV_BIN}}/python prepare_files.py {{n}} 
+	rm -rf tables_scale_{{n}}/*.tbl
+
+@run_private name $SCALE_FACTOR="1":
+	{{VENV_BIN}}/python -m {{name}}_queries.executor 
+
+run_glaredb $SCALE_FACTOR="1": 
+	just run_private "glaredb" $SCALE_FACTOR
+run_polars $SCALE_FACTOR="1": 
+	just run_private "polars" $SCALE_FACTOR
+run_duckdb $SCALE_FACTOR="1": 
+	just run_private "duckdb" $SCALE_FACTOR
+
+run n_runs="1" sf="1": requirements
+  #!/usr/bin/env bash
+  n=0
+  while [ $n -lt {{n_runs}}  ]; do
```

Comment:
We're running these tests in gpd,gpd,gpd order rather than ggg,ppp,ddd order. does this matter? Do they do something to flush ram buffers between runs?

I'd be inclined, also when running in CI to have each target 

I think I'd be inclined to have a "special" target named "all" that ran all three suites, and otherwise just have `run` with `suite` and `scale` all do the same thing.

Reply:
my first inclination also with this was that "we could surely do something with xargs", and it looks like `seq | xargs -I{} <cmd>` is the best option, and while I think this is fine in practice it is a bit weird. 

There's also `gnu parallel` which isn't installed everywhere, and is named oddly for given that we'd run it serially, just as a fancy loop, but that works, also I have learned that `zsh` has a `repeat` helper, because of course.

Reply:
should answer why interleave each execution environment rather than run one n times, and then run another?